torchdynamo/variables/nn_module.py

```diff
@@ -115,11 +115,15 @@ def var_getattr(self, tx, name):
 
         if name in base_dict:
             subobj = base_dict[name]
-        elif name in base_dict["_modules"] and name not in all_class_attribute_names:
```

Comment:
JIT-ed NN Module do not have these attr, so instead of a keyerror we shuffle on to let inspect. getattr_static handle it (or fail with a better message).